grpc-api/src/main/java/io/deephaven/grpc_api/console/ConsoleServiceGrpcImpl.java

```diff
@@ -49,6 +52,24 @@ public ConsoleServiceGrpcImpl(final Map<String, Provider<ScriptSession>> scriptT
         this.sessionService = sessionService;
         this.logBuffer = logBuffer;
         this.liveTableMonitor = liveTableMonitor;
+
+        if (!scriptTypes.containsKey(WORKER_CONSOLE_TYPE)) {
+            throw new IllegalArgumentException("console type not found: " + WORKER_CONSOLE_TYPE);
+        }
+
+        globalSession = scriptTypes.get(WORKER_CONSOLE_TYPE).get();
+    }
+
+    @Override
+    public void getConsoleTypes(final GetConsoleTypesRequest request,
```

Comment:
First time, give all session types. Once a session established, give the one we have started. Use same check when validating in startConsole.
Document this (and removing the WORKER_CONSOLE_TYPE plumbing?) in a TODO tied to a ticket to get the UI for session type selection correct.

Reply:
This is ticket #702. I left it in `triage` because I am uncertain of priority w.r.t. other UI tasks.